*** /usr/share/perl5/Smokeping.pm.old	2020-06-28 20:08:19.744877576 +0200
--- /usr/share/perl5/Smokeping.pm	2020-06-28 20:10:43.308089413 +0200
***************
*** 1879,1885 ****
                  $gotalert = $match unless $gotalert;
              my $edgetrigger = $alert->{edgetrigger} eq 'yes';
              my $what;
!             if ($edgetrigger and $prevmatch != $match) {
                  $what = ($prevmatch == 0 ? "was raised" : "was cleared");
              }
              if (not $edgetrigger and $match) {
--- 1879,1885 ----
                  $gotalert = $match unless $gotalert;
              my $edgetrigger = $alert->{edgetrigger} eq 'yes';
              my $what;
!             if ($edgetrigger and ($prevmatch == 0 ? 0 : 1) != $match) {
                  $what = ($prevmatch == 0 ? "was raised" : "was cleared");
              }
              if (not $edgetrigger and $match) {
